content_scripts/courseScheduale.js

```diff
@@ -105,8 +105,8 @@ const getMinMaxTimes = (data) => {
 }
 
 const addCourseScheduale = () => {
-    let courseContent = document.getElementsByClassName("course-content");
-    if (courseContent) {
+    const url = new URL(window.location.href)
```

Comment:
there was an exception that we get on the page before login because course content exists there.
I've changed it to check query params if it's having an id like in courseRemover